tests/snapshots/tests.contrib.anthropic.test_anthropic.test_anthropic_llm_sync.json

```diff
@@ -0,0 +1,40 @@
+[[
+  {
+    "name": "anthropic.request",
+    "service": "",
+    "resource": "anthropic.resources.messages.Messages",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "llm",
+    "error": 0,
+    "meta": {
+      "_dd.p.dm": "-0",
+      "_dd.p.tid": "66575bd600000000",
+      "_ml_obs.meta.input.messages": "[{\"content\": \"Can you explain what Descartes meant by 'I think, therefore I am'?\", \"role\": \"user\"}]",
+      "_ml_obs.meta.metadata": "{\"temperature\": 0.0, \"max_tokens\": 1024}",
+      "_ml_obs.meta.model_name": "claude-3-opus-20240229",
+      "_ml_obs.meta.output.messages": "[{\"content\": \"The famous philosophical statement \\\"I think, therefore I am\\\" (originally in Latin as \\\"Cogito, ergo sum\\\") was coined by the French philosopher Ren\\u00e9 Descartes in his work \\\"Discourse on the Method\\\" (1637). It is a foundational element of his philosophy of radical doubt.\\n\\nDescartes sought to find a solid foundation for knowledge by doubting everything that could be doubted. He reasoned that he could doubt the existence of his body, the external world, and even mathematical truths. However, he realized that he could not doubt the existence of his own mind and his own thinking. Even in the act of doubting, he was thinking.\\n\\nFrom this, Descartes concluded that the one thing he could be certain of was his own existence as a thinking being. Even if an evil demon were deceiving him about everything else, it could not deceive him about the fact that he was thinking, and therefore, he must exist.\\n\\nIn other words, \\\"I think, therefore I am\\\" means that the very act of thinking is proof of one's existence. It establishes self-awareness and consciousness as the bedrock of all other knowledge. This principle became a starting point for Descartes' further philosophical explorations and had a significant influence on the development of Western philosophy, particularly in the areas of epistemology (the study of knowledge) and metaphysics (the study of the nature of reality).\"}]",
```

Comment:
Just an FYI, these hidden tags should not be present in these snapshot files (should disappear when you uncomment the `llmobs_is_pc_sampled(...)` check